spec/schemas/cat.indices.yaml

```diff
@@ -29,13 +29,13 @@ components:
           type: string
         docs.count:
           description: available docs
-          oneOf:
+          anyOf:
             - type: string
             - nullable: true
               type: string
```

Comment:
probably scope creeping but `nullable` is not a proper JSON schema keyword. Even if it was, this is redundant because nullable string already encapsulates non-nullable string.

In this case and a few below, we can repalce `anyOf: ...` with `type: [string, null]` according to [this](https://json-schema.org/understanding-json-schema/reference/type)

Reply:
I tried this but it failed with some other error. Care to open an issue with some details to replace?